### PR TITLE
New package: system-monitoring-center 2.14.0

### PR DESCRIPTION
**Pull request by zen0bit**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x64-glib)

### CODE REVIEW

File: srcpkgs/system-monitoring-center/template [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#archs="i686 x86_64"
+#wrksrc=
+#create_wrksrc=yes
+build_style=python3-module
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="python3-setuptools python3-pip"
```
Comment (paper42) [inline srcpkgs/system-monitoring-center/template:14]: pip is not allowed

File: srcpkgs/system-monitoring-center/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#makedepends=""
+depends="dmidecode gtk+3 iproute2 python3-cairo python3-gobject eudev util-linux"
+short_desc="Multi-featured system monitor"
+maintainer=" <zen@osowoso.xyz>"
```
Comment (paper42) [inline srcpkgs/system-monitoring-center/template:18]: this is missing a name/nickname

File: srcpkgs/system-monitoring-center/template [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'system-monitoring-center'
+pkgname=system-monitoring-center
+version=1.35.1
+revision=1
+#archs="i686 x86_64"
+#wrksrc=
+#create_wrksrc=yes
```
Comment (paper42) [inline srcpkgs/system-monitoring-center/template:7]: all of these comments should be removed

**Comment by zen0bit** [comment]
In upstream repo:

# Dependencies
- These dependencies are already installed on many systems: 
    - `dmidecode, iproute2, python3 (>=3.6), python3-cairo, python3-gi, python3-gi-cairo, udev, util-linux`
- Following dependencies may be required on some systems:
    - `libcairo2-dev` (for systems with .deb packages)
    - `polkit` (for Arch Linux)
- Optional dependencies:
    - `x11-xserver-utils` or `xorg-xrandr` (for more accurate screen resolution and refresh rate detection)
    - `vcgencmd` (for physical RAM size, GPU frequency and video memory information on Raspberry Pi devices)

may I add something other?

for me everything looks good..

Also for network want NetworkManager running.
 I am not sure if is belongs to dependencies...

Center can run without it with quote about it so easily solvable...

**Comment by zen0bit** [comment]
Adopted to gtk4 ...

File: srcpkgs/system-monitoring-center/template [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'system-monitoring-center'
+pkgname=system-monitoring-center
+version=2.4.1
+revision=1
+build_style=python3-module
+hostmakedepends="pkg-config python3-cairo-devel python3-gobject-devel python3-setuptools"
+depends="dmidecode eudev libadwaita iproute2 python3-cairo python3-gobject util-linux"
+checkdepends="python3-build python3-pip python3-wheel"
```
Comment (paper42) [inline srcpkgs/system-monitoring-center/template:8]: there are no tests and these dependencies are wrong, please remove them

**Comment by zen0bit** [comment]
ready for review

**Comment by Duncaen** [comment]
Zero dependencies seems wrong.

**Pull request merged**